waspc/src/Wasp/Generator/Setup.hs

```diff
@@ -36,3 +44,12 @@ setUpDatabase spec dstDir sendMessage = do
   (dbGeneratorWarnings, dbGeneratorErrors) <- DbGenerator.postWriteDbGeneratorActions spec dstDir
   when (null dbGeneratorErrors) (sendMessage $ Msg.Success "Database successfully set up.")
   return (dbGeneratorWarnings, dbGeneratorErrors)
+
+buildSdk :: Path' Abs (Dir ProjectRootDir) -> Msg.SendMessage -> IO ([GeneratorWarning], [GeneratorError])
+buildSdk projectRootDir sendMessage = do
+  sendMessage $ Msg.Start "Building SDK..."
+  SdkGenerator.buildSdk projectRootDir >>= \case
+    Left errorMesage -> return ([], [GenericGeneratorError errorMesage])
```

Comment:
@Martinsos Is this alright as far as reporting errors is concerned? Should we do something extra?